rest/route/select.go

```diff
@@ -74,13 +76,36 @@ func (t *selectTestsHandler) Parse(ctx context.Context, r *http.Request) error {
 }
 
 func (t *selectTestsHandler) Run(ctx context.Context) gimlet.Responder {
-	// TODO (DEVPROD-11879): remove this log once test ROI MVP no longer needs
-	// this to debug inputs.
-	grip.Debug(message.Fields{
-		"message": "received test selection request",
-		"route":   "/rest/v2/select/tests",
-		"request": t.selectTests,
-		"ticket":  "DEVPROD-11629",
-	})
-	return gimlet.NewJSONResponse(t.selectTests)
+	tssBaseURL := t.env.Settings().TestSelection.URL
+	if tssBaseURL == "" {
+		return gimlet.NewJSONResponse(t.selectTests)
+	}
+
+	httpClient := utility.GetHTTPClient()
+	defer utility.PutHTTPClient(httpClient)
+	conf := testselection.NewConfiguration()
+	conf.HTTPClient = httpClient
+	conf.Servers = testselection.ServerConfigurations{
+		testselection.ServerConfiguration{
+			URL:         tssBaseURL,
+			Description: "Test selection service",
+		},
+	}
+	c := testselection.NewAPIClient(conf)
+	reqBody := testselection.BodySelectTestsApiTestSelectionSelectTestsProjectIdRequesterBuildVariantNameTaskIdTaskNamePost{
```

Comment:
That's a crazy name for a struct

Reply:
It's auto-generated OpenAPI code, so yeah, it's kinda crazy.